cutils.c

```diff
@@ -1201,6 +1204,110 @@ int64_t js__gettimeofday_us(void) {
     return ((int64_t)tv.tv_sec * 1000000) + tv.tv_usec;
 }
 
+#if defined(_WIN32)
+int js_exepath(char* buffer, size_t* size_ptr) {
```

Comment:
Can I suggest fixing up the style (right-leaning star, no blanks between cast and expression, that sort of thing)?

Reply:
Sure!